lib/testLauncher/SuitestLauncher.js

```diff
@@ -37,12 +37,17 @@ class SuitestLauncher {
 				process.exit(1);
 			}
 			const queue = new Queue(this.ownArgv.concurrency);
+			const consoleInfo = console.info;
```

Comment:
I'm not particularly fond of this solution. We are using our own logger anyway. Isn't it better to make it aware of execution context?

I would make a single logger module, that would export either logger or launcherLogger depending on context. And this one module would be used everywhere in the process. Similar how in React app on Swiss project we have store and store.dev and choose one based on env variable.

Reply:
ok